Documentation/Administration/SystemSettings/MaintenanceMode/Index.rst

```diff
@@ -0,0 +1,87 @@
+:navigation-title: Maintenance Mode
+..  include:: /Includes.rst.txt
+..  _maintenance-mode:
+
+=======================================================
+Maintenance mode: Prevent backend logins during upgrade
+=======================================================
+
+Set the backend into maintenance mode to prevent editors or even all
+administrators and CLI tools to access the TYPO3 backend.
+
+The maintenance mode is useful to prevent changes to the content during
+`Patch/Bugfix update <https://docs.typo3.org/permalink/t3coreapi:minor>`_
+and `Major upgrade <https://docs.typo3.org/permalink/t3coreapi:major>`_,
+database backups or in any other case where you want to prevent backend
```

Comment:
While creating backups nothing will be written to database. So, no modifications. Do you mean database restores instead?

Reply:
If you do a backup while someone is editing one record might already be changed (like the sys-file) while another is not yet (tt_content) and then you have an inconsistent data state